packages/resources/src/RemixSite.ts

```diff
@@ -0,0 +1,1506 @@
+import * as s3 from "aws-cdk-lib/aws-s3";
+import * as iam from "aws-cdk-lib/aws-iam";
+import * as logs from "aws-cdk-lib/aws-logs";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import * as route53 from "aws-cdk-lib/aws-route53";
+import * as s3Assets from "aws-cdk-lib/aws-s3-assets";
+import * as cloudfront from "aws-cdk-lib/aws-cloudfront";
+import * as acm from "aws-cdk-lib/aws-certificatemanager";
+import { AwsCliLayer } from "aws-cdk-lib/lambda-layer-awscli";
+import * as origins from "aws-cdk-lib/aws-cloudfront-origins";
+import * as route53Targets from "aws-cdk-lib/aws-route53-targets";
+import * as route53Patterns from "aws-cdk-lib/aws-route53-patterns";
+import { getChildLogger } from "@serverless-stack/core";
+const logger = getChildLogger("RemixSite");
+
+import { App } from "./App.js";
+import { Stack } from "./Stack.js";
+import { SSTConstruct } from "./Construct.js";
+import {
+  BaseSiteDomainProps,
+  BaseSiteReplaceProps,
+  BaseSiteCdkDistributionProps,
+  BaseSiteEnvironmentOutputsInfo,
+  getBuildCmdEnvironment,
+  buildErrorResponsesForRedirectToIndex,
+} from "./BaseSite.js";
+import { Permissions, attachPermissionsToRole } from "./util/permission.js";
+import * as crossRegionHelper from "./nextjs-site/cross-region-helper.js";
+
+const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
+
+type RemixConfig = {
+  assetsBuildDirectory: string;
+  publicPath: string;
+  serverBuildPath: string;
+  serverBuildTarget: string;
+  server?: string;
+};
+
+export interface RemixDomainProps extends BaseSiteDomainProps {}
+export interface RemixCdkDistributionProps
+  extends BaseSiteCdkDistributionProps {}
+export interface RemixSiteProps {
+  cdk?: {
+    /**
+     * Pass in bucket information to override the default settings this
+     * construct uses to create the CDK Bucket internally.
+     */
+    bucket?: s3.BucketProps;
+    /**
+     * Pass in a value to override the default settings this construct uses to
+     * create the CDK `Distribution` internally.
+     */
+    distribution?: RemixCdkDistributionProps;
+    /**
+     * Override the default CloudFront cache policies created internally.
+     */
+    cachePolicies?: {
+      /**
+       * Override the CloudFront cache policy properties for browser build files.
+       */
+      buildCachePolicy?: cloudfront.ICachePolicy;
+      /**
+       * Override the CloudFront cache policy properties for "public" folder
+       * static files.
+       *
+       * Note: This will not include the browser build files, which have a seperate
+       * cache policy; @see `buildCachePolicy`.
+       */
+      staticsCachePolicy?: cloudfront.ICachePolicy;
+      /**
+       * Override the CloudFront cache policy properties for responses from the
+       * server rendering Lambda.
+       * 
+       * @note The default cache policy that is used in the abscene of this property
+       * is one that performs no caching of the server response.
+       */
```

Comment:
**suggestion:** Perhaps it is worth noting that should the server response include a `Cache-Control` header then it will be included in the CloudFront cache?